### ISSUE TITLE
【EESS】New submissions for Tue,  4 Jul 23

### ISSUE THREAD

**Issue by Yukeaaa** [Yukeaaa/arxiv-daily#378]
## Keyword: volume render
There is no result 
## Keyword: volumetric render
There is no result 
## Keyword: remote render
There is no result 
## Keyword: hybrid render
There is no result 
## Keyword: raycast
There is no result 
## Keyword: medical imaging
There is no result 
## Keyword: medical visualization
There is no result 
## Keyword: interactive volume
There is no result 
## Keyword: rendering
There is no result 
## Keyword: cinematic rendering
There is no result 
## Keyword: volume data
There is no result 
## Keyword: remote visualization
There is no result 
## Keyword: direct volume rendering
There is no result 
## Keyword: mobile device
There is no result 
## Keyword: transfer function
There is no result 
## Keyword: retrieval
There is no result 
## Keyword: video retrieval
There is no result 
## Keyword: mobile
There is no result 
## Keyword: smartphone
There is no result 
## Keyword: medical volume data
There is no result 
## Keyword: webgpu
There is no result 
## Keyword: webgl
There is no result 
## Keyword: pre-rendering
There is no result 
## Keyword: prerendering
There is no result 
## Keyword: motion prediction
There is no result 
## Keyword: incremental learning
There is no result 
## Keyword: svm incremental
There is no result 
## Keyword: nerf
There is no result 
## Keyword: multiorgan
There is no result 
## Keyword: multi-organ
There is no result 
## Keyword: multi organ
There is no result 
### VoxWatch: An open-set speaker recognition benchmark on VoxCeleb
 - **Authors:** Raghuveer Peri, Seyed Omid Sadjadi, Daniel Garcia-Romero
 - **Subjects:** Audio and Speech Processing (eess.AS); Artificial Intelligence (cs.AI); Machine Learning (cs.LG)
 - **Arxiv link:** https://arxiv.org/abs/2307.00169
 - **Pdf link:** https://arxiv.org/pdf/2307.00169
 - **Abstract**
 Despite its broad practical applications such as in fraud prevention, open-set speaker identification (OSI) has received less attention in the speaker recognition community compared to speaker verification (SV). OSI deals with determining if a test speech sample belongs to a speaker from a set of pre-enrolled individuals (in-set) or if it is from an out-of-set speaker. In addition to the typical challenges associated with speech variability, OSI is prone to the "false-alarm problem"; as the size of the in-set speaker population (a.k.a watchlist) grows, the out-of-set scores become larger, leading to increased false alarm rates. This is in particular challenging for applications in financial institutions and border security where the watchlist size is typically of the order of several thousand speakers. Therefore, it is important to systematically quantify the false-alarm problem, and develop techniques that alleviate the impact of watchlist size on detection performance. Prior studies on this problem are sparse, and lack a common benchmark for systematic evaluations. In this paper, we present the first public benchmark for OSI, developed using the VoxCeleb dataset. We quantify the effect of the watchlist size and speech duration on the watchlist-based speaker detection task using three strong neural network based systems. In contrast to the findings from prior research, we show that the commonly adopted adaptive score normalization is not guaranteed to improve the performance for this task. On the other hand, we show that score calibration and score fusion, two other commonly used techniques in SV, result in significant improvements in OSI performance.
### Digital Filters for Instantaneous Frequency Estimation
 - **Authors:** Hugh Lachlan Kennedy
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.00452
 - **Pdf link:** https://arxiv.org/pdf/2307.00452
 - **Abstract**
 This technical note is on digital filters for the high-fidelity estimation of a sinusoidal signal's frequency in the presence of additive noise. The complex noise is assumed to be white (i.e. uncorrelated) however it need not be Gaussian. The complex signal is assumed to be of (approximately) constant magnitude and (approximately) polynomial phase such as the chirps emitted by bats, whale songs, pulse-compression radars, and frequency-modulated (FM) radios, over sufficiently short timescales. Such digital signals may be found at the end of a sequence of analogue heterodyning (i.e. mixing and low-pass filtering), down to a bandwidth that is matched to an analogue-to-digital converter (ADC), followed by digital heterodyning and sample rate reduction (optional) to match the clock frequency of the processor. The spacing of the discrete frequency bins (in cycles per sample) produced by the Fast Fourier Transform (FFT) is equal to the reciprocal of the window length (in samples). However, a long FFT (for fine frequency resolution) has a high complexity and a long latency, which may be prohibitive in embedded closed-loop systems, and unnecessary when the channel only contains a single sinusoid. In such cases, and for signals of constant frequency, the conventional approach involves the (weighted) average of instantaneous phase differences. General, naive, optimal, and pragmatic (recursive), filtering solutions are discussed and analysed here using Monte-Carlo (MC) simulations.
### Generalized NOMP for Line Spectrum Estimation and Detection from  Coarsely Quantized Samples
 - **Authors:** Jiang Zhu, Hansheng Zhang, Ning Zhang, Zhiwei Xu, Jun Fang
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.00491
 - **Pdf link:** https://arxiv.org/pdf/2307.00491
 - **Abstract**
 As radar systems accompanied by large numbers of antennas and scale up in bandwidth, the cost and power consumption of high-precision (e.g., 10-12 bits) analog-to-digital converter (ADC) become the limiting factor. As a remedy, line spectral estimation and detection (LSE\&D) from low resolution (e.g., 1-4 bits) quantization has been gradually drawn attention in recent years. As low resolution quantization reduces the dynamic range (DR) of the receiver, the theoretical detection probabilities for the multiple targets (especially for the weakest target) are analyzed, which reveals the effects of low resolution on weak signal detection and provides the guidelines for system design. The computation complexities of current methods solve the line spectral estimation from coarsely quantized samples are often high. In this paper, we propose a fast generalized Newtonized orthogonal matching pursuit (GNOMP) which has superior estimation accuracy and maintains a constant false alarm rate (CFAR) behaviour. Besides, such an approach are easily extended to handle the other measurement scenarios such as sign measurements from time-varying thresholds, compressive setting, multisnapshot setting, multidimensional setting and unknown noise variance. Substantial numerical simulations are conducted to demonstrate the effectiveness of GNOMP in terms of estimating accuracy, detection probability and running time. Besides, real data are also provided to demonstrate the effectiveness of the GNOMP.
### Robust Target Localization in 2D: A Value-at-Risk Approach
 - **Authors:** João Domingos, João Xavier
 - **Subjects:** Signal Processing (eess.SP); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2307.00548
 - **Pdf link:** https://arxiv.org/pdf/2307.00548
 - **Abstract**
 This paper consider considers the problem of locating a two dimensional target from range-measurements containing outliers. Assuming that the number of outlier is known, we formulate the problem of minimizing inlier losses while ignoring outliers. This leads to a combinatorial, non-convex, non-smooth problem involving the percentile function. Using the framework of risk analysis from Rockafellar et al., we start by interpreting this formulation as a Value-at-risk (VaR) problem from portfolio optimization. To the best of our knowledge, this is the first time that a localization problem was formulated using risk analysis theory. To study the VaR formulation, we start by designing a majorizer set that contains any solution of a general percentile problem. This set is useful because, when applied to a localization scenario in 2D, it allows to majorize the solution set in terms of singletons, circumferences, ellipses and hyperbolas. Using know parametrization of these curves, we propose a grid method for the original non-convex problem. So we reduce the task of optimizing the VaR objective to that of efficiently sampling the proposed majorizer set. We compare our algorithm with four benchmarks in target localization. Numerical simulations show that our method is fast while, on average, improving the accuracy of the best benchmarks by at least 100m in a 1 Km$^2$ area.
### A multi-task learning framework for carotid plaque segmentation and  classification from ultrasound images
 - **Authors:** Haitao Gan, Ran Zhou, Yanghan Ou, Furong Wang, Xinyao Cheng, Xiaoyan Wu, Aaron Fenster
 - **Subjects:** Image and Video Processing (eess.IV); Computer Vision and Pattern Recognition (cs.CV)
 - **Arxiv link:** https://arxiv.org/abs/2307.00583
 - **Pdf link:** https://arxiv.org/pdf/2307.00583
 - **Abstract**
 Carotid plaque segmentation and classification play important roles in the treatment of atherosclerosis and assessment for risk of stroke. Although deep learning methods have been used for carotid plaque segmentation and classification, most focused on a single task and ignored the relationship between the segmentation and classification of carotid plaques. Therefore, we propose a multi-task learning framework for ultrasound carotid plaque segmentation and classification, which utilizes a region-weight module (RWM) and a sample-weight module (SWM) to exploit the correlation between these two tasks. The RWM provides a plaque regional prior knowledge to the classification task, while the SWM is designed to learn the categorical sample weight for the segmentation task. A total of 1270 2D ultrasound images of carotid plaques were collected from Zhongnan Hospital (Wuhan, China) for our experiments. The results of the experiments showed that the proposed method can significantly improve the performance compared to existing networks trained for a single task, with an accuracy of 85.82% for classification and a Dice similarity coefficient of 84.92% for segmentation. In the ablation study, the results demonstrated that both the designed RWM and SWM were beneficial in improving the network's performance. Therefore, we believe that the proposed method could be useful for carotid plaque analysis in clinical trials and practice.
### Model-Assisted Probabilistic Safe Adaptive Control With Meta-Bayesian  Learning
 - **Authors:** Shengbo Wang, Ke Li, Yin Yang, Yuting Cao, Tingwen Huang, Shiping Wen
 - **Subjects:** Systems and Control (eess.SY); Machine Learning (cs.LG); Optimization and Control (math.OC)
 - **Arxiv link:** https://arxiv.org/abs/2307.00828
 - **Pdf link:** https://arxiv.org/pdf/2307.00828
 - **Abstract**
 Breaking safety constraints in control systems can lead to potential risks, resulting in unexpected costs or catastrophic damage. Nevertheless, uncertainty is ubiquitous, even among similar tasks. In this paper, we develop a novel adaptive safe control framework that integrates meta learning, Bayesian models, and control barrier function (CBF) method. Specifically, with the help of CBF method, we learn the inherent and external uncertainties by a unified adaptive Bayesian linear regression (ABLR) model, which consists of a forward neural network (NN) and a Bayesian output layer. Meta learning techniques are leveraged to pre-train the NN weights and priors of the ABLR model using data collected from historical similar tasks. For a new control task, we refine the meta-learned models using a few samples, and introduce pessimistic confidence bounds into CBF constraints to ensure safe control. Moreover, we provide theoretical criteria to guarantee probabilistic safety during the control processes. To validate our approach, we conduct comparative experiments in various obstacle avoidance scenarios. The results demonstrate that our algorithm significantly improves the Bayesian model-based CBF method, and is capable for efficient safe exploration even with multiple uncertain constraints.
### Scenario Based Cost Optimization of Water Distribution Networks Powered  by Grid-Connected Photovoltaic Systems
 - **Authors:** Mirhan Ürkmez, Carsten Kallesøe, Jan Dimon Bendtsen, John Leth
 - **Subjects:** Systems and Control (eess.SY)
 - **Arxiv link:** https://arxiv.org/abs/2307.00845
 - **Pdf link:** https://arxiv.org/pdf/2307.00845
 - **Abstract**
 The paper presents a predictive control method for the water distribution networks (WDNs) powered by photovoltaics (PVs) and the electrical grid. This builds on the controller introduced in a previous study and is designed to reduce the economic costs associated with operating the WDN. To account for the uncertainty of the system, the problem is solved in a scenario optimization framework, where multiple scenarios are sampled from the uncertain variables related to PV power production. To accomplish this, a day-ahead PV power prediction method with a stochastic model is employed. The method is tested on a high-fidelity model of a WDN of a Danish town and the results demonstrate a substantial reduction in electrical costs through the integration of PVs, with PVs supplying $66.95\%$ of the required energy. The study also compares the effectiveness of the stochastic optimization method with a deterministic optimization approach.
### Waveform and Beamforming Optimization for Wireless Power Transfer with  Dynamic Metasurface Antennas
 - **Authors:** Amirhossein Azarbahram (1), Onel L. A. Lopez (1), Bruno Clerckx (2), Matti Latva-Aho (1) ((1) Centre for Wireless Communications (CWC), University of Oulu, Finland, (2) Department of Electrical and Electronic Engineering, Imperial College London, London, U.K, and Silicon Austria Labs (SAL), Austria )
 - **Subjects:** Signal Processing (eess.SP)
 - **Arxiv link:** https://arxiv.org/abs/2307.01081
 - **Pdf link:** https://arxiv.org/pdf/2307.01081
 - **Abstract**
 Radio frequency (RF) wireless power transfer (WPT) is a promising charging technology for future wireless systems. However, low end-to-end power transfer efficiency (PTE) is a critical challenge for practical implementations. One of the main inefficiency sources is the power consumption and loss of key components such as the high-power amplifier (HPA) and rectenna, which must be considered for PTE optimization. Herein, we investigate the power consumption of an RF-WPT system considering the emerging dynamic metasurface antenna (DMA) as the transmitter. Moreover, we incorporate the HPA and rectenna non-linearities and consider the Doherty HPA to reduce power consumption. We provide a mathematical framework to calculate each user's harvested power from multi-tone signal transmissions and the system power consumption. Then, the waveform and beamforming are designed using swarm-based intelligence to minimize power consumption while satisfying the users' energy harvesting (EH) requirements. Numerical results manifest that increasing the number of transmit tones enhances the performance in terms of coverage probability and power consumption since the HPAs operate below the saturation region in the simulation setup and the EH non-linearity is the dominant factor. Finally, our findings demonstrate that a properly shaped DMA may outperform a fully-digital antenna of the same size.
